doc/backend/07adding_fields_to_entities.md

```diff
@@ -0,0 +1,40 @@
+# Steps to add a field to a model
+
+1. Create a migration  
+   `cd backend && diesel migration generate my_new_migration`  
+   This will create `up.sql` and `down.sql` inside a new migration folder under `/backend/migrations/`  
+   `up.sql` should create all necessary tables, columns, constraints as well as modify data if necessary  
+   `down.sql` should undo all changes from `up.sql`
+
+2. Run the migration  
+   `make migration`  
+   During development you can run `make migration-redo` to run `down.sql` and `up.sql` of the most recent migration
+
+3. Add new field to Entity  
+   e.g. extend the `Plants` entity in [entity.rs](https://blob.permaplant.net/master/backend/src/model/entity.rs)
```

Comment:
Do we need to rename this folder then? Or is it different as it is related to the architecture of the backend?

Reply:
That might be a leftover from the diesel getting started guide, where the term `model` is used - no mention of entities. I looked at a handful of other rust projects using diesel and all seem to be using the term `model`